src/main/java/io/jenkins/plugins/audit/listeners/ItemChangeListener.java

```diff
@@ -0,0 +1,137 @@
+package io.jenkins.plugins.audit.listeners;
+
+import hudson.Extension;
+import hudson.ExtensionList;
+import hudson.model.Item;
+import hudson.model.User;
+import io.jenkins.plugins.audit.event.*;
+import org.apache.logging.log4j.audit.LogEventFactory;
+
+import javax.annotation.Nonnull;
+import java.util.Date;
+
+/**
+ * Listener notified of any CRUD operations on items.
+ */
+@Extension
+public class ItemChangeListener extends hudson.model.listeners.ItemListener{
+
+    /**
+     * Fired when a new job is created and added to Jenkins, before the initial configuration page is provided.
+     *
+     * @param item is a job that is created
+     */
+    @Override
+    public void onCreated(@Nonnull Item item) {
+        CreateItem itemCreateEvent = LogEventFactory.getEvent(CreateItem.class);
+
+        itemCreateEvent.setItemName(item.getName());
+        itemCreateEvent.setItemUri(item.getUrl());
+        itemCreateEvent.setTimestamp(new Date().toString());
```

Comment:
This date format works for now, though we should certainly figure out a common format to use in all audit events. I'm a fan of ISO-8601 style formatting, though I'm not sure if we're consistent about that yet. Another task to write up.

Reply:
Thanks, I added this task to the JIRA board shared by you.